gravitee-gateway-core/src/main/java/io/gravitee/gateway/core/logging/LimitedLoggableClientRequest.java

```diff
@@ -17,22 +17,20 @@
 
 import io.gravitee.gateway.api.Request;
 import io.gravitee.gateway.api.buffer.Buffer;
+import io.gravitee.reporter.api.log.LogConfiguration;
```

Comment:
Why `LogConfiguration` is part of reporter-api ?
It seems to be really specific to the gateway, and is not tightly coupled to the reporter-api.

WDYT?

Reply:
I thought that the need of SSL Info logging, certs logging and max size of body to be logged relate directly to reporter-api (there earlier was only maxSizeLogMessage settings and now to not throw them through different call parameters they are united to LogConfiguration class). I'd leave it there - this is not directly gateway logging parameters.

Reply:
Let me think about it.... My side, I was telling myself that LogConfiguration  was only used by the gateway, which is doing report. Also the LogConfiguration is highly dependent on the config you've settled (from `gravitee.yml`)

Reply:
Also, you'll see that the ES reporter is never using this `LogConfiguration` class. So not sure reporter-api is the best place.

Please let me know what you think.

Reply:
well, you're right - |'ll move LogConfiguration to gateway package/module